README.md

```diff
@@ -40,4 +36,4 @@ it has been added because it cleans the header of Java files by removing mislead
 7. **MissingJavadocType rule is ignored on test files:** Usually test files do not require javadoc as it is obvious 
 what class this test is for.
 
-7. **Suppression file(if exists) must be located beside the main file**.
\ No newline at end of file
+7. **Suppression file(if exists) must be located beside the main file**.
```

Comment:
According to the new changes in the structure of this file, this section describes our code style customizations in general, not the exact changes in checkstyle.xml. This item is a minor change in the checkstyle.xml and is not about the changes in rules, so I think it is better not to mention it.